### ISSUE TITLE
Is DeepMosaics abandonware at this point?

### ISSUE THREAD

**Issue by Milincho** [HypoX64/DeepMosaics#101]
DeepMosaics v0.4.1 was released on Feb 07 2021, more than a year ago.

v0.5.0 and v0.5.1 were released 3 months later, in May 11 and 23 2021.

It's Feb 2022 now and despite the many users bug reports there hasn't been any news of new versions coming.

Not to sound ungrateful, DeepMosaics has been a great piece of software that just needs some fixes, and I would gladly pay for newer versions, but honestly, shouldn't we consider the project kind of abandoned at this point?

**Comment by Milincho** [comment]
I guess it is then...   :'(

**Comment by Mario-119** [comment]
Yeah it's unfortunate, it's really promising stuff but the GPU workload option seems to be broken on my (and others) end.

**Comment by Milincho** [comment]
> Yeah it's unfortunate, it's really promising stuff but the GPU workload option seems to be broken on my (and others) end.

Worse than that, there isn't even any response at all from the developer. Tried sending him emails also but never got any answer.

**Comment by seatv** [comment]
I would say that HypiX64 has a day job :) He seems to respond to some of the issues posted here, abandonware might be too strong a disposition. It would be nice to have someone setup a forum to discuss these topics and have a self support group. We can also put any models that folks have built so that others can benefit.

**Comment by Milincho** [comment]
I would definately pay Premium or send him donations or Patreon subscription if that's what would take to keep this project going on and getting updates.

**Comment by wind82** [comment]
Software is great it's just got some errors and is slow. I would love to know the process steps or get a new version that takes real advantage of the gpu. I started using javplayer and tg plus which is WORLDS faster but the result is not as good as deepmosaics 0.5.1 when set to something like THR4.

**Comment by Milincho** [comment]
6 months later and still not a single update or any kind of news at all...   :-(

Pity this has become abandonware so soon and before reaching its full potential.

**Comment by Boltzmachine** [comment]
Anyone who wants to Make It Great Again?
